conf/env/PRESTASHOP.env

```diff
@@ -16,6 +16,7 @@ PS_FOLDER_INSTALL=installOLD
 ADMIN_MAIL=demo@reverb.com
 ADMIN_PASSWD=123456
 PS_HANDLE_DYNAMIC_DOMAIN=0
+PS_ERASE_DB=1
```

Comment:
If you install the container the first Time, you must to configure this variable at 1, it's better if you want à database linked with prestashop. If you're container is stopped, you update this field at 0 and docker could be restart without data base installation.